diff --git a/src/layout/scrolling.rs b/src/layout/scrolling.rs
index 6a876910..2abea290 100644
--- a/src/layout/scrolling.rs
+++ b/src/layout/scrolling.rs
@@ -3176,7 +3176,7 @@ impl<W: LayoutElement> ScrollingSpace<W> {
                     && self.active_column_idx == col_idx
                     // In tabbed mode, all tabs have activated state to reduce unnecessary
                     // animations when switching tabs.
-                    && (active_in_column || is_tabbed);
+                    && (active_in_column);
                 win.set_activated(active);
 
                 win.set_interactive_resize(col_resize_data);
-- 
2.47.2

